cse-valid-adapter-app/src/main/java/com/farao_community/farao/cse_valid/adapter/app/listener/CseValidAdapterListener.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.cse_valid.adapter.app.listener;
+
+import com.farao_community.farao.cse_valid.adapter.app.service.CseValidAdapterService;
+import com.farao_community.farao.gridcapa.task_manager.api.TaskDto;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Component;
+import reactor.core.publisher.Flux;
+
+import java.util.function.Consumer;
+
+/**
+ * @author Oualid Aloui {@literal <oualid.aloui at rte-france.com>}
+ */
+@Component
+public class CseValidAdapterListener {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CseValidAdapterListener.class);
+    private final CseValidAdapterService cseValidAdapterService;
+    private static final Object LOCK = new Object();
+
+    public CseValidAdapterListener(CseValidAdapterService cseValidAdapterService) {
+        this.cseValidAdapterService = cseValidAdapterService;
+    }
+
+    @Bean
+    public Consumer<Flux<TaskDto>> consumeTask() {
+        return f -> f
+                .onErrorContinue((t, r) -> LOGGER.error(t.getMessage(), t))
+                .subscribe(this::handleRunTask);
```

Comment:
This is the part you should get inspired by the task-manager :
@Bean
    public Consumer<Flux<NotificationRecords>> consumeMinioEvent() {
        return f -> f.subscribe(nr -> {
            try {
                handleMinioEvent(nr);
            } catch (Exception e) {
                LOGGER.error("Unable to handle MinIO event properly", e);
            }
        });
    }
You can remove the onErrorContinue and implement a try catch handling.